src/app/sas.service.ts

```diff
@@ -14,7 +14,9 @@ export class SasService {
       serverUrl: '',
       appLoc: '/Public/app/angular',
       serverType: 'SASVIYA',
-      debug: true
+      debug: true,
+      useComputeApi: true,
+      contextName: 'SharedCompute'
```

Comment:
this context doesn't exist in a default environment - let's use "SAS Job Execution compute context" here instead